device/install.md

```diff
@@ -34,9 +34,10 @@
 
 ## Connecting to NAIS clusters
 1. In a terminal/shell of your choice, navigate to [kubeconfigs repo](https://github.com/navikt/kubeconfigs)'.
-   1. If you haven't downloaded repo already: `git clone <clone URL> && cd kubeconfigs`
+   1. If you haven't downloaded repo already: `git clone https://github.com/navikt/kubeconfigs.git && cd kubeconfigs`
```

Comment:
@jhrv Er ikke sikkert alle kan/vil bruke https. Forslag til hvordan å ikke få folk til å tro at de _bør_/_skal_ bruke https?

Reply:
Ja, kanskje punkt 1 og 2 her kan slås sammen til noe som "Make sure you have the latest [kubeconfigs repository](https://github.com/navikt/kubeconfigs)' on your computer (either cloning it from Github or a git pull To ensure you've got latest & greatest)." 
Jeg er usikker på om jeg gjorde punkt 3 (git checkout naisdevice Remember to use the right branch!), det kan jeg ikke huske å ha gjort... "Remember to use the right branch" trenger kanskje litt mer info, hvordan gjør man det?